test/unit/scan/test_scan_policies.py

```diff
@@ -7,14 +7,207 @@
 # License https://www.backblaze.com/using_b2_code.html
 #
 ######################################################################
+from pathlib import Path
+import platform
 import re
-
+from unittest.mock import MagicMock
+from b2sdk.scan.folder import LocalFolder
+from b2sdk.utils import fix_windows_path_limit
 import pytest
```

Comment:
imports are not grouped properly as `isort` would group them. As a convention in this project we group imports manually (should probably just use `isort` - I'm not sure why we don't have it set up right now, maybe it was clashing with yapf?)